Guides/Xenoarcheology/Xenoarch: Chapter 2

```diff
@@ -0,0 +1,126 @@
+
+# Xenoarcheology Guide - Chapter 2: Stimulus-Reaction Pairs
```

Comment:
This should be on the first line of the document to function as a document label. That aside, the text is too long to print fully as a label even when this change is made.

![image](https://github.com/user-attachments/assets/00adb1c5-14ea-4e48-8de5-8fe140d06e23)

This could be fixed by abbreviating certain words.
```suggestion
# Xenoarch Guide - Ch.2: Stimulus-Reaction Pairs
```
Result:
![image](https://github.com/user-attachments/assets/58f88279-d359-41ca-b0f7-bfb521484454)